path/path.go

```diff
@@ -116,15 +127,19 @@ func ParseCidToPath(txt string) (Path, error) {
 	return FromCid(c), nil
 }
 
+// IsValid returns true if the path is valid, and false otherwise.
```

Comment:
doesnt return true or false, it returns an error if its invalid, and nil otherwise